test/tools/yulopti.cpp

```diff
@@ -43,14 +43,27 @@
 
 #include <libsolutil/JSON.h>
 
+#include <boost/algorithm/string/predicate.hpp>
+#include <boost/algorithm/string/join.hpp>
 #include <boost/program_options.hpp>
 
+#include <range/v3/action/sort.hpp>
+#include <range/v3/range/conversion.hpp>
+#include <range/v3/view/concat.hpp>
+#include <range/v3/view/drop.hpp>
+#include <range/v3/view/map.hpp>
+#include <range/v3/view/set_algorithm.hpp>
+#include <range/v3/view/stride.hpp>
+#include <range/v3/view/transform.hpp>
```

Comment:
This is probably the only thing I don't like about ranges-v3. I didn't want to just bulk-include everything but figuring out individual files to include is a bit bothersome as the docs don't list the header where each function is defined.